release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -457,6 +457,11 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-release-notes-storage_{context}"]
 === Storage
 
+[id="ocp-4-17-storage-editing-overprovisioning-ratio"]
+==== Editing the over-provisioning ratio after creating the LVMCluster custom resource
```

Comment:
Since this is RN heading and not a procedure, you can use a heading something like below avoiding gerund:
```suggestion
==== Over-provisioning ratio update after LVMCluster custom resource creation
```

Reply:
Updated as per the suggestion.